.telescope.json

```diff
@@ -2,7 +2,7 @@
   "protoDirs": [
     "./protos"
   ],
-  "outPath": "./libs/cosmos-types",
+  "outPath": "./networks/injective/codegen-injective",
```

Comment:
the .telescope.json file in the project root is generated every time telescope runs. it only summarizes the config of the last run of telescope. so I think we should delete it and put it into gitignore.

Reply:
Good idea, deleted and added to gitignore